### PR TITLE
Add proper readme content

### PR DESCRIPTION
**Pull request by redbluenat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,53 @@
...
+- [Supported Platforms](installation-and-updates/supported-platforms.md)
 
+## App Navigation
 
+- [Interface](app-navigation/navigation.md)
+
+## Basics
+
+- [Create first project](basics/create-first-project.md)
+- [Sending Api Request](basics/sending-api-request.md)
```
Comment (MihalyBezzeg) [inline README.md]:
```suggestion
- [Mocking an API Request](basics/sending-api-request.md)
```

Comment (MihalyBezzeg) [inline README.md]: I'd use here the same title as on the opened page like for the others.

**Comment by giorgioprovenzale** [comment]
Looks good!

**Review by MihalyBezzeg — APPROVED** [review]
👍

**Pull request merged**